nautobot/extras/tests/test_filters.py

```diff
@@ -163,6 +165,19 @@ def test_search(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
 
+class ConfigContextFilterSetTestCase(TestCase):
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=True)
+    def test_with_dynamic_groups_enabled(self):
+        filter_set = ConfigContextFilterSet()
+        self.assertIsNotNone(filter_set.filters.get("dynamic_groups", None))
+        
+
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=False)
+    def test_without_dynamic_groups_enabled(self):
+        filter_set = ConfigContextFilterSet()
+        self.assertIsNone(filter_set.filters.get("dynamic_groups", None))
```

Comment:
No need for this to be a separate test case. If you could please just roll these methods into `ConfigContextTestCase` below that would be ideal. Also, docstrings on the `test_` methods, please! :)